### DISCUSSION TITLE
Unable to list all kafka topics in Kubernetes

### DISCUSSION THREAD

**Discussion by amclatav** [strimzi/strimzi-kafka-operator#8374] [Q&A]
Hi everyone,

I have been searching and searching online for some kind of help regarding this issue, but I haven't been lucky.

Since I created a new cluster in kubernetes, not all kafka topics are listed when I run the commands:
kubectl get kafkatopic
kubectl get strimzi

I have 2 kafka clusters set up and I can see all the kafka topics in one of the clusters, but not in the other one.

I know they exist because if I use Offset Explorer I can see them.

Has this happened to anyone?
Any tips?

Thank you :)

**Comment by scholzj** [comment]
Check if your Topic Operator is working and if not, what errors it is reporting.

**Reply by amclatav** [reply]
Thank you @scholzj your suggestion has helped me figuring out a solution for my issue.
